cyclonedx_py/client.py

```diff
@@ -235,6 +235,10 @@ def get_arg_parser(*, prog: Optional[str] = None) -> argparse.ArgumentParser:
             '-pb', '--purl-bom-ref', action='store_true', dest='use_purl_bom_ref',
             help="Use a component's PURL for the bom-ref value, instead of a random UUID"
         )
+        arg_parser.add_argument(
+            "-omit", "--omit", dest="omit", action="append",
```

Comment:
we do not want duplicate shorts/longs.
remove the `-omit`. just keep the `--omit`.

Reply:
Fixed - I fully agree. I did it like this because of the existing '-o', '--o',

Reply:
@madpah what do you thing? 
should ge tackle this in the CLI revirite that is possibly upcoming? 

How would you like to have the args/switches?

Reply:
I'm happy to accept this PR before we look to migrate to `click8` - it's a great addition and don't want to burden @tngraf with the additional pain!